projects/igniteui-angular/src/lib/data-operations/filtering-expressions-tree.ts

```diff
@@ -96,14 +101,28 @@ export class FilteringExpressionsTree implements IFilteringExpressionsTree {
 
     public returnFields?: string[];
 
+    /**
+     * Array of filtering operand providers to be used when generating type hints
+     * during serialization.
+     * @memberof FilteringExpressionsTree
+     */
+    public expressionTypes? = [
```

Comment:
Expression types are all serialized with null. Maybe we should serialize the names?
![image](https://github.com/user-attachments/assets/b78fa581-d902-4591-be75-fa6e104a5891)

Reply:
I've removed the `expressionTypes` field from the serialized model.

Reply:
I've changed `expressionTypes` to be static so the are not present on the model, and do not get lost during serialization.